lua/telescope/make_entry.lua

```diff
@@ -685,14 +685,30 @@ function make_entry.gen_from_buffer_lines(opts)
     separator = ' │ ',
     items = {
       { width = 5 },
-      { remaining = true },
+      { dynamic_highlight = true, },
```

Comment:
Is that actually a thing? I think you solved this differently so we should change that line back to not confuse people reading that code :)

Reply:
well technically, I don't think `remaining = true` does anything either haha

Reply:
Good call.
Just thinking out load: We should probably make it explicit and error on wrong key input, so people aren't up being confused if they have a typo, when using `width` or `right_justify` or something sunjon wanted some time ago, was right side align, not column but result side (currently not possible because we need to lazy evaluate the result window width).

For me it makes sense to validate the items tbl.